### PR TITLE
feat: [IOCOM-717] Update error message in the `MessageRouterScreen`

### PR DESCRIPTION
**Pull request by adelloste**
## Short description
This PR updates the error message in the `MessageRouterScreen` if `getThirdPartyMessage` fails.

## List of changes proposed in this pull request
- updated error message in the `MessageRouterScreen`
- updated locales

## How to test
Using `io-dev-api-server` generate a message with remote content that fails. Open the message detail, check that the error message is correctly displayed

### CODE REVIEW

File: ts/sagas/messages/watchThirdPartyMessageSaga.ts [hunk 1]
```diff
@@ -59,7 +59,9 @@ function* getThirdPartyMessage(
         loadThirdPartyMessage.success({ id, content: thirdPartyMessage })
       );
     } else {
-      const reason = `Response status ${result.right.status}`;
+      const reason = `Response status ${result.right.status} - ${
+        result.right.value?.detail || "UNKNOWN"
```
Comment (Vangaorth) [inline ts/sagas/messages/watchThirdPartyMessageSaga.ts:63]: What about `no "detail" field provided` instead of "UNKNOWN"?

**Review by Vangaorth — CHANGES_REQUESTED** [review]

File: ts/sagas/messages/watchThirdPartyMessageSaga.ts (hunk 1 above)
Comment (adelloste) [reply ts/sagas/messages/watchThirdPartyMessageSaga.ts]: Applied https://github.com/pagopa/io-app/pull/5318/commits/7939171193c40283b703d5b4fe8df5b4d59577bc

**Review by Vangaorth — APPROVED** [review]
LGTM

File: locales/en/index.yml
Comment (adelloste) [inline locales/en/index.yml:1]: missing translations

**Pull request merged**